### ISSUE TITLE
Helm chart for fluentd

### ISSUE THREAD

**Issue by cameel** [golemfactory/concent-deployment#224]
This task depends on #208 and supersedes #40.

The goal of this task is to prepare a Helm chart that deploys a DaemonSet running fluentd with a custom config file in a ConfigMap that will allow us to adjust parsing rules.

The idea of of the task is similar to #40 but we want to have Helm deploy the DeamonSet instead of doing it manually.

- Create a Helm chart for fluentd in `kubernetes/concent-fluentd/` directory in the repository.
    - Include a Kubernetes manifest for the DaemonSet
        - It should be based on the manifest that's deployed by default by GKE
        - See [helm/charts/incubator/fluentd-cloudwatch/](https://github.com/helm/charts/tree/master/incubator/fluentd-cloudwatch) for ideas on how you may need to customize a manifest for deployment using Helm.
    - Include a ConfigMap that contains the default configuration
        - Don't focus on the configuration in this task. We want the defaults. There will be a separate task for adding custom rules.
        - The configuration should be downloaded from an external repository by the Makefile
    - Make the most important values configurable via `values.yaml`.
    - If the chart needs access to secrets, it should depend on the chart created in #206.
- Modify `kubernetes/Makefile` so that it builds the chart
- Modify the deployment playbook so that it installs or updates the chart on the cluster
    - It can run in the same namespace as all the other services.

When this is merged, reconfigure clusters to disable the built-in fluentd and run ours instead.

Verify that logs in Stackdriver stay the same and we're not losing anything.

It may be useful to implement #225 first so that you can easily remove everything from the cluster.

**Comment by PaweuB** [comment]
Pivotal:
https://www.pivotaltracker.com/story/show/160169408
